package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "remote-settings-mdn-browser-data",
```

Comment:
Should this match the name of the repo? `remote-settings-mdn-browser-compat-data`

Reply:
sure